site/content/docs/commands/deploy.en.md

```diff
@@ -5,33 +5,57 @@ $ copilot deploy
 
 ## What does it do?
 
-This command is used to run either [`copilot svc deploy`](../commands/svc-deploy.en.md) or [`copilot job deploy`](../commands/job-deploy.en.md) under the hood. The steps involved in `copilot deploy` are the same as those involved in `copilot svc deploy` and `copilot job deploy`:
+This command is used to run either [`copilot svc deploy`](../commands/svc-deploy.en.md) or [`copilot job deploy`](../commands/job-deploy.en.md) under the hood.
 
-1. When `image.build` exists in the manifest:
+It also checks for service and environment manifests in the local workspace, and allows you to initialize and deploy workloads and environments.
+
+If a workload is uninitialized, `--init-wkld` will initialize the workload before deploying it.
+
+If the desired environment is uninitialized, you may initialize it with `--init-env`. 
+
+The `--deploy-env` flag can be specified to skip environment deployment confirmation, or can be set to false (`--deploy-env=false`) to skip 
+deploying the environment.
+
+The steps involved in `copilot deploy` are the same as those involved in `copilot svc deploy` and `copilot job deploy`:
+
+1. If your service does not exist, optionally initialize it (via `--init-env[=false]`).
+2. If the target environment does not exist, optionally initialize it with custom credentials (via `--init-env[=false]`, `--region`, `--profile`, or `--aws-*` flags).
+3. Optionally deploy the environment before service deployment (via `--deploy-env[=false]`).
```

Comment:
actually i'm wondering if we should just not show the flags in this numbered section 🤔 we mentioned them above and below already, and we follow the logic whether or not the flags are present, just prompting instead.